tests/test_pretest.py

```diff
@@ -60,28 +62,33 @@ def collect_dut_info(dut):
     return { 'intf_status' : status }
 
 
-def test_update_testbed_metadata(duthosts, tbinfo):
+def test_update_testbed_metadata(duthosts, dut_hostname, tbinfo):
```

Comment:
This function is ready for multi-dut. you don't have to change this one. Please back out your change.

Reply:
ok, doesn't hurt to split either.

Reply:
Yeah, let's put the original code back. We need to generate a file with information from all duts. Enumerating dut_hostname is not right for this function.

Reply:
The new code also generated one file with info from both DUTs. However the new code had to play with json file twice to handle this. Original code was safer and simpler. Reverted this set of code, thanks.